### PR TITLE
Support 1MB SRAM feature under qemu

### PR DESCRIPTION
**Pull request by junmin-kim**
User can use 1MB SRAM by patching qemu-2.10.0-rc2_increase_ram_size.patch
To use 16MB SRAM, user should patch qemu-2.10.0-rc2_maximize_ram_size.patch

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/qemu/README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ Targets Supported:
 NOTE:  
 Tizen RT supports two memory options for lm3s6963-ek board;
```
Comment (sunghan-chang) [inline build/configs/qemu/README.md]: two -> three

File: build/configs/qemu/tc_1m/Make.defs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+############################################################################
+# build/configs/qemu/tc_16m/Make.defs
```
Comment (lokeshbv) [inline build/configs/qemu/tc_1m/Make.defs]: change tc_16m ---> tc_1m

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**